File: 70-climbing-stairs/70-climbing-stairs.js
/**
 * @param {number} n
 * @return {number}
 */
var climbStairs = function(n) {
        let a=0,b=1;
        let c=a+b;
        a=b;
        b=c;
        if(n==1)
            return c;
        n--;
        while(n>0){
            c=a+b;
            a=b;
            b=c;
            n--;
        }
        return c;
};

File: 70-climbing-stairs/70-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
           int a=0,b=1;
        int c=a+b;
        a=b;
        b=c;
        if(n==1)
            return c;
        n--;
        while(n>0){
            c=a+b;
            a=b;
            b=c;
            n--;
        }
        return c;
    }
};